.gitignore

```diff
@@ -75,7 +75,6 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
 fastlane/test_output
-fastlane/readme.md
```

Comment:
why you remove the fastlane readme? this is not the readme that you have to use

Reply:
Solved!